tests/integration/targets/azure_rm_virtualmachinescaleset/tasks/main.yml

```diff
@@ -41,7 +43,9 @@
   azure_rm_loadbalancer:
     resource_group: "{{ resource_group }}"
     name: testLB1
-    public_ip_address_name: testPublicIP1
+    frontend_ip_configurations:
+      - name: frontendipconf0
+        public_ip_address: testPublicIP1
```

Comment:
Add backend_address_pools. Before the deletion, backend_address_pools has a default value if it is not configured
```suggestion
        public_ip_address: testPublicIP1
      backend_address_pools:
        - name: backendaddrp0
```